config/initializers/secret_token.rb

```diff
@@ -6,7 +6,5 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-Safecast::Application.config.secret_token =
-  '39c00ee18c6a2edd472f8f8b5954b6a332636e32b4c80ed9abc9053cc33bf7f1913d4cb6267ea741d10b48acfd9b3b3df2336fd02de83a4290509bc5d97c0edd'
 Safecast::Application.config.secret_key_base =
-  'f0906ce1f97f1d3a7b1fba927645046ef10d9f7766d2141e023dc074529d9e2b10294e9a6490b04aff9d95ce4764c65e3280a30e1dd2e94f37ba22381ec2b4a4'
+  '39c00ee18c6a2edd472f8f8b5954b6a332636e32b4c80ed9abc9053cc33bf7f1913d4cb6267ea741d10b48acfd9b3b3df2336fd02de83a4290509bc5d97c0edd'
```

Comment:
This is a security problem and we should fix it soon after doing the upgrade. I created https://github.com/Safecast/safecastapi/issues/665